src/vivarium_census_prl_synth_pop/components/businesses.py

```diff
@@ -123,15 +131,17 @@ def on_initialize_simulants(self, pop_data: SimulantData) -> None:
         """
         if pop_data.creation_time < self.start_time:
             # Initial population setup
-            self.businesses = self.generate_businesses(pop_data)
+            self.businesses = self.generate_businesses()
 
         pop = self.population_view.subview(["age", "household_id"]).get(
             pop_data.index, query="tracked"
         )
 
         pop["employer_id"] = data_values.Unemployed.EMPLOYER_ID
         working_age = pop.loc[pop["age"] >= data_values.WORKING_AGE].index
-        pop.loc[working_age, "employer_id"] = self.assign_random_employer(working_age)
+        pop.loc[working_age, "employer_id"] = self.assign_random_employer(
+            working_age, randomness_stream=self.business_moves_randomness
```

Comment:
Now that I've PRed this I'm not convinced we want to use the CRN here.

Reply:
I don't think we want to use it here.

Reply:
We definitely don't want to use this here. We should use the standard randomness stream. The business moves randomness stream should be used only for initializing and modifying the businesses data structure. 

Honestly, assigning employers should really happen in a different component altogether, as it's really an entirely different task, but that refactor is definitely out of scope here.